sridha2412/assignment2/q1.py

```diff
@@ -0,0 +1,36 @@
+# print ancestors of given key in a binary tree
+
+# node class`
+class Node:
+    # constuctor to initlalize a node object
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+# if key is present, print ancestors else return false
```

Comment:
Please set these comments as "docstrings":
https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments

def print_ancestors(root, key):
  """If key is present, print ancestors else return false."""